symfony.lock

```diff
@@ -136,41 +130,26 @@
     "doctrine/sql-formatter": {
         "version": "1.1.2"
     },
-    "ergebnis/json-printer": {
-        "version": "3.2.0"
-    },
     "friendsofphp/proxy-manager-lts": {
         "version": "v1.0.5"
     },
-    "guzzlehttp/guzzle": {
-        "version": "7.4.2"
-    },
-    "guzzlehttp/promises": {
-        "version": "1.5.1"
-    },
     "guzzlehttp/psr7": {
         "version": "2.1.0"
     },
-    "http-interop/http-factory-guzzle": {
-        "version": "1.2.0"
-    },
     "jetbrains/phpstorm-stubs": {
         "version": "v2019.3"
     },
     "justinrainbow/json-schema": {
         "version": "5.2.12"
     },
     "knplabs/github-api": {
-        "version": "3.6",
+        "version": "3.7",
```

Comment:
any reason for this update?

Reply:
it is. `composer update` for specific package or for all packages. 
Nice to have an obvious explanation instead of the answer.